src/transformers/models/opt/tokenization_opt_fast.py

```diff
@@ -0,0 +1,303 @@
+# coding=utf-8
```

Comment:
Let's delete this file

The tokenizer is a GPT2-like tokenizer

Reply:
We can reuse its code

Reply:
resolved in b931db8efbe7e79c7da2114017fe587265e2007e